util/exec/exec.go

```diff
@@ -36,7 +37,8 @@ func Run(cmd *exec.Cmd) (string, error) {
 func RunWithRedactor(cmd *exec.Cmd, redactor func(text string) string) (string, error) {
 	span := tracing.NewLoggingTracer(log.NewLogrusLogger(logrus.New())).StartSpan(fmt.Sprintf("exec %v", cmd.Args[0]))
 	span.SetBaggageItem("dir", fmt.Sprintf("%v", cmd.Dir))
-	span.SetBaggageItem("args", fmt.Sprintf("%v", cmd.Args))
+	args := regexp.MustCompile("(--username|--password) [^ ]*").ReplaceAllString(fmt.Sprintf("%v", cmd.Args), "$1 ******")
```

Comment:
I think `redactor` parameter already knows how to reduce sensitive data. So the fix is rather 👍 
```
span.SetBaggageItem("args", redactor(fmt.Sprintf("%v", cmd.Args)))
```

Reply:
Yes.